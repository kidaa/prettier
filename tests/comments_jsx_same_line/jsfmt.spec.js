run_spec(__dirname, { jsxBracketSameLine: true }, ["babylon", "typescript"]);
