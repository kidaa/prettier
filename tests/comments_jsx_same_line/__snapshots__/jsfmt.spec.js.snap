// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jsx_same_line.js 1`] = `
<div
  // comment
>
  {foo}
</div>;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div
// comment
>
  {foo}
</div>;

`;
